strawberry/static/graphiql.html

```diff
@@ -32,6 +32,7 @@
     <script src="https://cdn.jsdelivr.net/npm/graphiql-with-extensions@0.14.3/graphiqlWithExtensions.min.js"
         integrity="sha384-TqI6gT2PjmSrnEOTvGHLad1U4Vm5VoyzMmcKK0C/PLCWTnwPyXhCJY6NYhC/tp19"
         crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/js-cookie@rc/dist/js.cookie.min.js"></script>
```

Comment:
Is there a particular reason why you're using the v3 release candidate and not the stable v2 version?

Reply:
Just grabbed the CDN link from https://github.com/js-cookie/js-cookie#cdn.

Reply:
Ok fair enough!